Makefile

```diff
@@ -3,7 +3,7 @@ release:
 	rm -rf dist
 	rm -rf build
 	scripts/git_tag.sh
-	python setup.py sdist bdist_wheel
+	python -m build -nwsx
```

Comment:
I don't think this will work... What this target did was get the version by parsing `__init__.py`, create a git tag of the form `v{version}` (e.g., `v0.4.0`), then build the package and upload it to PyPi. I would call this after running `bumpversion.sh` to update the package version in the code. 

I think your workflow is the opposite: first create a git tag, then use that tag as the package version when building and uploading the package.